src/swap/components/SwapProvider.tsx

```diff
@@ -95,6 +94,10 @@ export function SwapProvider({
 
   const { from, to } = useFromTo(address);
 
+  const { sendTransactionAsync } = useSendTransaction();
```

Comment:
Can we start adding some line comments on what each thing is for.... otherwise this is going to get hard to mantain.

Reply:
done